### PR TITLE
Infrastructure python scripts

### PR DESCRIPTION
**Pull request by mike1729**

### CODE REVIEW

**Review by timorl — CHANGES_REQUESTED** [review]
There is a dire need for a readme explaining how to use this (incl. ofc the required authorization files and whatnot).

**Review by fbielejec — APPROVED** [review]
looks good, but indeed could use a README

File: README.md [hunk 1]
```diff
@@ -1,2 +1,16 @@
...
-Aleph Zero Blockchain Helm chart with all the necessary tooling to run it locally.
+
+Aleph Zero Blockchain python scripts for running TestNet and DevNet.
+
+# Usage instructions
+
+- Create an account on AWS, set up credentials, and a default region as described [here](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/quickstart.html#configuration).
+- Put keys for ssh in key_pairs.
+- Install packages needed for orchestrating experiments: GNU parallel, zip, unzip and Python 3 packages: fabric, boto3, ipython, tqdm, matplotlib.
+- Then, `run ipython -i shell.py`. This opens a shell with procedures orchestrating experiments.
```
Comment (timorl) [inline README.md:10]: The `run` shouldn't be part of the command.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,16 @@
...
+  The main procedures are `run_protocol(n_processes, regions, instance_type)` and
+  `run_devnet(n_processes, regions, instance_type)` that run `n_processes` spread
```
Comment (timorl) [inline README.md:12]: A minimal working example would be nice. I think it's `run_devnet(4)`, but I'm not sure.

Comment (timorl) [reply README.md:12]: It would also be nice to know how to turn this off. <____<

**Review by timorl — CHANGES_REQUESTED** [review]
Minor manual comments. I wasn't able to run this yet.

File: docker/docker-compose.yml [hunk 3]
```diff
@@ -7,10 +7,10 @@ services:
...
     volumes:
       - /var/run/docker.sock:/var/run/docker.sock
-    environment:
-      - WATCHTOWER_NOTIFICATIONS=slack
-      - WATCHTOWER_NOTIFICATION_SLACK_HOOK_URL=$SLACK_HOOK_URL
-      - WATCHTOWER_NOTIFICATION_SLACK_IDENTIFIER=Devnet-Watchtower-Service
+    # environment:
+    #   - WATCHTOWER_NOTIFICATIONS=slack
+    #   - WATCHTOWER_NOTIFICATION_SLACK_HOOK_URL=$SLACK_HOOK_URL
+    #   - WATCHTOWER_NOTIFICATION_SLACK_IDENTIFIER=Devnet-Watchtower-Service
```
Comment (timorl) [inline docker/docker-compose.yml:13]: Commented code, just in case someone misses it. <_<"

Comment (mike1729) [reply docker/docker-compose.yml]: Removed.

**Pull request merged**